Examples/NewModuleTemplate/plasterManifest.xml

```diff
@@ -53,6 +53,10 @@
         </parameter>
     </parameters>
     <content>
+        <message>
+Scaffold a PowerShell Module with the files required to run Pester tests, build with PSake and publish to the PSGallery.
+
```

Comment:
Should we bake the ending newline into the code so that templates don't need to add one?